lib/setup.php

```diff
@@ -61,14 +61,10 @@ function setup() {
  */
 function widgets_init() {
 
-	// There is an issue where the opening `.widget__content` div is not
-	// included if the widgets doesn't have a title. If we can't find a way to
-	// reliably wrap the widget content in this `.widget__content` div we'll
-	// probably need to remove it and find another way to style the widgets.
 	$defaults = [
 		'before_widget' => '<section class="widget--boxed %1$s %2$s">',
 		'before_title'  => '<h3 class="widget__title">',
-		'after_title'   => '</h3><div class="widget__content">',
+		'after_title'   => '</h3>',
```

Comment:
i'm referring to the above 2-3 lines with the after_title still containing hr closing tag, which doesn't get executed if there's no title.